app/components/Forms/AltDateInput.tsx

```diff
@@ -0,0 +1,352 @@
+// eslint-disable
```

Comment:
If we don't figure out some way to properly disable eslint here, it turns out there is a laundry list of fails that must be fixed manually, some of which seem to trigger a chain of required updates:
```
  components/Forms/AltDateInput.tsx:1:0
  ⚠  110:5   Unexpected todo comment.                                                                                                                                                                                                                                                                                      no-warning-comments
  ⚠  292:1   Function deepEquals has a complexity of 30. Maximum allowed is 20.                                                                                                                                                                                                                                            complexity
  ✖    1:0   Specify the rules you want to disable.                                                                                                                                                                                                                                                                        unicorn/no-abusive-eslint-disable
  ✖   52:14  Operands of + operation must either be both strings or both numbers. Consider using a template literal.                                                                                                                                                                                                       @typescript-eslint/restrict-plus-operands
  ✖   52:14  Operands of + operation must either be both strings or both numbers. Consider using a template literal.                                                                                                                                                                                                       @typescript-eslint/restrict-plus-operands
  ✖   98:5   State initialization should be in a class property                                                                                                                                                                                                                                                            react/state-in-constructor
  ✖  101:3   componentWillReceiveProps is deprecated since React 16.9.0, use UNSAFE_componentWillReceiveProps instead, see https://reactjs.org/docs/react-component.html#unsafe_componentwillreceiveprops. Use https://github.com/reactjs/react-codemod#rename-unsafe-lifecycles to automatically update your components.  react/no-deprecated
  ✖  184:24  Do not use Array index in keys                                                                                                                                                                                                                                                                                react/no-array-index-key
  ✖  198:11  Unexpected negated condition.                                                                                                                                                                                                                                                                                 no-negated-condition
  ✖  207:11  Unexpected negated condition.                                                                                                                                                                                                                                                                                 no-negated-condition
  ✖  250:21  Operands of + operation must either be both strings or both numbers. Consider using a template literal.                                                                                                                                                                                                       @typescript-eslint/restrict-plus-operands
  ✖  265:3   Parameter time is declared but its value is never read.                                                                                                                                                                                                                                                       @typescript-eslint/no-unused-vars-experimental
  ✖  362:3   Require compare argument.                                                                                                                                                                                                                                                                                     @typescript-eslint/require-array-sort-compare
  ✖  363:3   Require compare argument.                                                                                                                                                                                                                                                                                     @typescript-eslint/require-array-sort-compare
```

For example if I tackle `componentWillReceiveProps` by changing it to `UNSAFE_componentWillReceiveProps`, when I run xo again it then says: 
```
"UNSAFE_componentWillReceiveProps is unsafe for use in async rendering. Update the component to use getDerivedStateFromProps instead. See https://reactjs.org/blog/2018/03/27/update-on-async-rendering.html"
```
...And then running xo again tells me:
```
UNSAFE_componentWillReceiveProps is unsafe for use in async rendering. Update the component to use getDerivedStateFromProps instead. See https://reactjs.org/blog/2018/03/27/update-on-async-rendering.html.  react/no-unsafe
```
which isn't exactly a straightforward update.

Reply:
Let me look into disabling eslint

Reply:
For reference, [this was the PR](https://github.com/sindresorhus/eslint-plugin-unicorn/pull/308) that led me to believe we should be able to do something like: 
`// eslint-disable unicorn/no-abusive-eslint-disable`
or
`// eslint-disable unicorn/no-abusive-eslint-disable react/no-deprecated`

Reply:
I can report marginal success using: `/* eslint-disable react/no-deprecated */` where you only list one single rule that you're unwilling to fix

Reply:
If that works then let's go with this, I'll add a card in the issue tracker about using a better date/time input